fortran/testpar/multidsetrw.F90

```diff
@@ -156,6 +156,7 @@ SUBROUTINE pmultiple_dset_hyper_rw(do_collective, do_chunk, mpi_size, mpi_rank,
   CALL h5dwrite_multi_f(ndsets, dset_id, mem_type_id, mem_space_id, file_space_id, buf_md, error, plist_id)
   CALL check("h5dwrite_multi_f", error, nerrors)
 
+  return
```

Comment:
Mismerge or intended here?

Reply:
Matches code in develop, but looks out of place?

Reply:
@brtnfld I see return is a statement in Fortran. Was this meant to be here?

Reply:
It's correct. It's not really needed, but it does not hurt anything.